lamindb_setup/core/_hub_crud.py

```diff
@@ -77,6 +77,18 @@ def select_instance_by_id(
     return response.data[0]
 
 
+def select_instance_by_id_with_storage(
+    instance_id: str,
+    client: Client,
+):
+    response = (
+        client.table("instance").select("*, storage(*)").eq("id", instance_id).execute()
```

Comment:
This may give problems because you're not specifying the foreign key constraint - I'm not sure though 🤔

Reply:
Right, it doesn't seem needed for storage, only for account. All good.